SOUTH AFRICA - ITP | Luke Manyamazi | Module-User-Focused-Data | Week 1

Luke-Manyamazi opened the pull request:
<!--

You must title your PR like this:

COHORT_NAME | FIRST_NAME LAST_NAME | REPO_NAME | WEEK

For example,

NW4 | Carol Owen | HTML-CSS-Module | Week1

Complete the task list below this message.
If your PR is rejected, check the task list.

-->

## Learners, PR Template

Self checklist

- [* ] I have committed my files one by one, on purpose, and for a reason
- [* ] I have titled my PR with COHORT_NAME | FIRST_NAME LAST_NAME | REPO_NAME | WEEK 
- [* ] I have tested my changes
- [* ] My changes follow the [style guide](https://syllabus.codeyourfuture.io/guides/code-style-guide/)
- [* ] My changes meet the [requirements](./README.md) of this task

## Changelist
- 

Briefly explain your PR.
This is a request for the reviewer to check my html code and css code of the Git page that I created.

## Questions

Ask any questions you have for your reviewer.
On the changes list, am I supposed to list every commit that I have pushed to github?

Luke-Manyamazi committed:
Added a closing html tag

Luke-Manyamazi committed:
Added html markup

Luke-Manyamazi committed:
Remove old css and add new

Luke-Manyamazi committed:
Added css

Luke-Manyamazi committed:
Add footer to html

Luke-Manyamazi committed:
add footer css

Luke-Manyamazi committed:
Removed border guides

Luke-Manyamazi committed:
Removed margin-top on the header and added background -color to the footer

Luke-Manyamazi committed:
Remove task comments

eediallo reviewed (changes requested):
Hi @Luke-Manyamazi ! Well done on successfully raising a PR for this project! It’s a really good start.

You can preview your work here: https://deploy-preview-149--cyf-ufd.netlify.app/wireframe/

I’ve added some suggestions for your attention.

Also, the title of your PR is almost correct, but you need to change the first part. Instead of "ITP," it should be the cohort name, such as "Cape Town" or "London."

eediallo commented on Wireframe/index.html:
```diff
@@ -1,13 +1,76 @@
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8" />
```
The trailling slash here has no effect and interacts badly with unquoted attribute values:
```suggestion
 <meta charset="UTF-8" >

eediallo replied:
The trailing slash on void element has no effect and iteracts badly with unquoted attributes values:
```suggestion
 <meta charset="UTF-8" >

eediallo replied:
The trailling slash has no effect here
```suggestion
  <meta charset="UTF-8" >

eediallo commented on Wireframe/index.html:
```diff
@@ -1,13 +1,76 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" type="text/css" href="./css/styles.css" />
```
The trailling slash here has no effect and interacts badly with unquoted attribute values:
```suggestion
 <meta name="viewport" content="width=device-width, initial-scale=1.0" >

eediallo replied:
The trailing slash on void element has no effect and iteracts badly with unquoted attributes values:
```suggestion
  <link rel="stylesheet" type="text/css" href="./css/styles.css" >

eediallo replied:
The trailling slash has no effect here
```suggestion
 <link rel="stylesheet" type="text/css" href="./css/styles.css" >

eediallo commented on Wireframe/index.html:
```diff
@@ -1,13 +1,76 @@
+          Subversion, CVS, Perforce, and ClearCase with features like cheap
+          local branching, convenient staging areas, and multiple workflows.
+        </p>
+        <button>
+          <a href="https://git-scm.com/" target="_blank">READ MORE</a>
+        </button>
```
The element a must not appear as a descendant of the button element.
```suggestion
 <a href="https://git-scm.com/" target="_blank">READ MORE</a>

eediallo commented on Wireframe/index.html:
```diff
@@ -1,13 +1,76 @@
+            centralized version control system. It even makes better use of
+            junior developers by giving them a safe environment to work in.
+          </p>
+          <button>
+            <a
+              href="https://www.atlassian.com/git/tutorials/why-git"
+              target="_blank"
+            >
+              READ MORE</a
+            >
+          </button>
```
The element a must not appear as a descendant of the button element.
```suggestion
  <a href="https://www.atlassian.com/git/tutorials/why-git" target="_blank" >
```
You can style the a tag so that it looks like a button

eediallo commented on Wireframe/index.html:
```diff
@@ -1,13 +1,76 @@
+            how big or how small—you spawn a new branch to encapsulate your
+            changes.
+          </p>
+          <button>
+            <a
+              href="https://www.w3schools.com/git/git_branch.asp?remote=github"
+              target="_blank"
+              >READ MORE</a
+            >
+          </button>
```
The element a must not appear as a descendant of the button element.
```suggestion
<a href="https://www.w3schools.com/git/git_branch.asp?remote=github" target="_blank"  >READ MORE</a
```
You can style the a tag so that it looks like a button

Luke-Manyamazi committed:
Update Wireframe/index.html

Co-authored-by: Elhadj Abdoul  Diallo <elhhabdouldiallo@gmail.com>

Luke-Manyamazi committed:
Update Wireframe/index.html

Co-authored-by: Elhadj Abdoul  Diallo <elhhabdouldiallo@gmail.com>

Luke-Manyamazi commented:
> Hi @Luke-Manyamazi ! Well done on successfully raising a PR for this project! It’s a really good start.
> 
> You can preview your work here: https://deploy-preview-149--cyf-ufd.netlify.app/wireframe/
> 
> I’ve added some suggestions for your attention.
> 
> Also, the title of your PR is almost correct, but you need to change the first part. Instead of "ITP," it should be the cohort name, such as "Cape Town" or "London."

Thank you!

Commit:
removed / on the closing link and meta tags

Commit:
Merge branch 'feature/wireframe' of https://github.com/Luke-Manyamazi/Module-User-Focused-Data into feature/wireframe

Commit:
remove button elements

Commit:
delete button css

Commit:
added css for <a> element

Commit:
added padding to the paragraphs

Commit:
added padding to the footer element

Commit:
added images to articles

Luke-Manyamazi commented:
Hi @eediallo, I have edited my index.html and CSS files and incorporated your great suggestions. I also added images to each article. Kindly review.

eediallo commented on Wireframe/index.html:
```diff
@@ -1,13 +1,76 @@
+      </header>
+
+      <section class="article1">
+        <img class="article1_img" src="/Wireframe/git2.png">
```
img element must have an alt attribute, except under certain conditions. 
```suggestion
<img class="article1_img" src="/Wireframe/git2.png" alt"adds image description here">
```

More about the img element here: https://www.w3.org/WAI/tutorials/images/

eediallo commented on Wireframe/index.html:
```diff
@@ -1,13 +1,76 @@
+
+      <div class="section-container">
+          <section class="article">
+            <img class="article2_img" src="/Wireframe/git1.png">
```
img element must have an alt attribute, except under certain conditions. 
```suggestion
<img class="article1_img" src="/Wireframe/git1.png" alt"adds image description here">
```

More about the img element here: https://www.w3.org/WAI/tutorials/images/

eediallo commented on Wireframe/index.html:
```diff
@@ -1,13 +1,76 @@
+          
+        </section>
+        <section class="article">
+          <img class="article3_img" src="/Wireframe/git3.png">
```
img element must have an alt attribute, except under certain conditions. 
```suggestion
<img class="article1_img" src="/Wireframe/git3.png" alt"adds image description here">
```

More about the img element here: https://www.w3.org/WAI/tutorials/images/

eediallo commented on Wireframe/index.html:
```diff
@@ -1,13 +1,76 @@
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
The trailing slash has no effect here.
```suggestion
 <meta name="viewport" content="width=device-width, initial-scale=1.0" >

eediallo reviewed (changes requested):
Hi @Luke-Manyamazi . Well done for making the recommended changes. 

I have made some final recommendations for your attention. I think you will be fine after you have made these changes. 

Next time I recommend you use a **HTML validator** like this[ one](https://validator.w3.org/nu/) to check you code. It will give you an idea of where needs fixing in your code.

Commit:
removed trailing hashes on elements and added alt attributes to all img elements

Luke-Manyamazi commented:
Thank you, @eediallo I have removed the trailing hashes and also added the alt attribute to the img elements. Thank you for the HTML checker, I used it to check my code and everything looks fine now. See the 
![Screenshot 2024-10-16 at 11 36 12](https://github.com/user-attachments/assets/205fee9e-d714-4764-a8de-d45e6e8db2a3)
attached screenshot of the results.

Luke-Manyamazi committed:
added space on color css
